File: Monstone_Android_Client/app/src/main/java/com/example/nightwingky/monstone/main_program/product_Item/ProductHttp.java
package com.example.nightwingky.monstone.main_program.product_Item;

import com.example.nightwingky.monstone.myConst.URLConst;

import java.io.IOException;

import okhttp3.FormBody;
import okhttp3.OkHttpClient;
import okhttp3.Request;
import okhttp3.Response;

/**
 * Created by nightwingky on 17-3-20.
 */

public class ProductHttp {

    public static String addToCart(String no, String cart) throws IOException {
        OkHttpClient okHttpClient = new OkHttpClient();

        FormBody formBody = new FormBody.Builder()
                .add("no", no)
                .build();

        Request request = new Request.Builder()
                .url(URLConst.getAdd_to_shopping_cart())
                .post(formBody)
                .build();

        Response response = okHttpClient.newCall(request).execute();

        if(response.isSuccessful()) {
            return response.body().string();
        } else {
            return null;
        }
    }
}

File: kotlin_client/app/src/main/java/com/graduation/nightwingky/book_drifting/monstone/main_program/productItem/ProductHttp.kt
package com.graduation.nightwingky.book_drifting.monstone.main_program.productItem

import com.graduation.nightwingky.book_drifting.monstone.myConst.URLConst

import java.io.IOException

import okhttp3.FormBody
import okhttp3.OkHttpClient
import okhttp3.Request
import okhttp3.Response

/**
 * Created by nightwingky on 17-3-20.
 */

object ProductHttp {

    @Throws(IOException::class)
    fun addToCart(no: String, cart: String): String? {
        val okHttpClient = OkHttpClient()

        val formBody = FormBody.Builder()
                .add("no", no)
                .add("cart", cart)
                .build()

        val request = Request.Builder()
                .url(URLConst.add_to_shopping_cart)
                .post(formBody)
                .build()

        val response = okHttpClient.newCall(request).execute()

        return if (response.isSuccessful) {
            response.body()!!.string()
        } else {
            null
        }
    }
}
